NanaZip.UI.Modern/SevenZip/CPP/7zip/UI/FileManager/PanelListNotify.cpp

```diff
@@ -92,6 +95,62 @@ UString ConvertSizeToString(UInt64 value)
   return s;
 }
 
+
+
+// **************** NanaZip Modification Start **************
+static std::wstring ConvertSizeToByteUnitString(
+    std::uint64_t ByteSize)
+{
+    const wchar_t* Units[] =
+    {
+        L"Byte",
+        L"Bytes",
+        L"KiB",
+        L"MiB",
+        L"GiB",
+        L"TiB",
+        L"PiB",
+        L"EiB"
+    };
+    const std::size_t UnitsCount = sizeof(Units) / sizeof(*Units);
+
+    // Output Format:
+    // For ByteSize is 0 or 1: x Byte
+    // For ByteSize is from 2 to 1023: x Bytes
+    // For ByteSize is larger than 1023: x.xx {KiB, MiB, GiB, TiB, PiB, EiB}
+
+    std::size_t UnitIndex = 0;
+    double Result = static_cast<double>(ByteSize);
+
+    if (ByteSize > 1)
+    {
+        for (UnitIndex = 1; UnitIndex < UnitsCount; ++UnitIndex)
+        {
+            if (1024.0 > Result)
+                break;
+
+            Result /= 1024.0;
+        }
+
+        // Keep two digits after the decimal point.
+        Result = static_cast<std::uint64_t>(Result * 100) / 100.0;
+    }
+
+    return std::wstring(Mile::FormatWideString(
+        (UnitIndex > 1) ? L"%.2f %s" : L"%.0f %s",
+        Result,
+        Units[UnitIndex]));
+}
+static void ConvertSizeToByteUnitString(UInt64 value,wchar_t* String)
```

Comment:
This definition code style is not OK. Must be:
```
static void ConvertSizeToByteUnitString(
    UInt64 Value,
    wchar_t* String)
```

NEVER BE LAZY!

There are several similar issues.